tests/spec/fs.write.spec.js

```diff
@@ -1,24 +1,25 @@
-var util = require('../lib/test-utils.js');
-var expect = require('chai').expect;
+'use strict';
+const util = require('../lib/test-utils.js');
+const expect = require('chai').expect;
 
 describe('fs.write', function() {
   beforeEach(util.setup);
   afterEach(util.cleanup);
 
   it('should be a function', function() {
-    var fs = util.fs();
+    let fs = util.fs();
     expect(fs.write).to.be.a('function');
   });
 
   it('should error if file path is undefined', function() {
-    var fs = util.fs();
-    var fn = () => fs.writeFile(undefined, 'data');
+    let fs = util.fs();
+    let fn = () => fs.writeFile(undefined, 'data');
```

Comment:
fn can be const as well
```suggestion
    const fn = () => fs.writeFile(undefined, 'data');
```